protos/symbol.proto

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+message ModuleSymbols {
+  uint64 load_bias = 1;
+  string symbols_file_path = 2;
+  repeated SymbolInfo symbol_infos = 3;
+}
+
+message SymbolInfo {
+  string name = 1;
+  string pretty_name = 2;
+  uint64 address = 3;
+  uint64 size = 4;
+  string source_file = 5;
```

Comment:
Are the line and source_file info now displayed in the Functions tab?

Reply:
@ronaldfw , this is good timing, we have the first step towards having a heat map in the source code view.

Reply:
The line is not displayed yet. This is work that still needs to be done. (its only in the proto, because we know we want it). Compare: ElfFileImpl::LoadSymbols(); currenlty ElfFile.cpp::157

Reply:
The file is actually set to OrbitFunction::File(), which is currently set to the file that contains the symbols (module file or .debug file)

Reply:
Oh ok I see :-).  I guess it shouldn't be too hard to grab that info when parsing symbols right?  It would help us out for a new feature.  Also, it will probably contribute to bigger transfer sizes.

Reply:
I also dont think that should be too hard. But it should be a different PR. There is at least b//154580143 and maybe others for that